pkg/policyengine/examples/allowAll.json

```diff
@@ -14,4 +13,4 @@
             }
         ]
     }
-}
+}
```

Comment:
nit: missing newline

Reply:
Fixed. Apparently VSCode has a setting for this: `files.insertFinalNewline`. Default is Off, which means it will remove final new lines whenever a file gets saved.